### ISSUE TITLE
code of merit is not meritocratic 

### ISSUE THREAD

**Issue by driusan** [rosarior/Code-of-Merit#31]
Many of the points in the code of merit relate to judging contributors and giving power based on quality of past contributions. This is the opposite of meritocracy, where contributions would be judged independently of who contributed them and without bias based on the value of the contribution itself.

**Comment by EvgenijM86** [comment]
You confuse judgement of contributions with the judgement of the contributor. It's not like Code states that contributions should be automatically considered better just because they came from a contributor with better history.

**Comment by tpruzina** [comment]
Depends on what we consider "contribution", e.g. is every commit a contribution? Is every PR a contribution? Or perhaps overall contribution shall be also considered - this includes past contribution(s).

If we strive for ideal of meritocracy, in the end, most projects form a hierarchy of competence (in bigger projects a cyclic graph forming the "web of trust"). These structures inherently model the above when peer review of each contribution by all contributors becomes unsustainable. 

I don't think those two are necessarily incompatible.

**Comment by driusan** [comment]
@herrold I'm saying that the many of the points in the "code of merit" such as, apparently, the most important one:

> 1. The project creators, lead developers, core team, constitute the managing members of the project and have final say in every decision of the project, technical or otherwise, including overruling previous decisions. There are no limitations to this decisional power.

or others such as:

>  4. Authority or position in the project will be proportional to the accrued contribution. Seniority must be earned.

serve to re-enforce the existing power dynamic of a project and give some contributors' contributions more weight than others, which is counter to judging individual contributions based on their own merit regardless of who made them (as would be the assumed goal of a "code of merit").

**Comment by Russtopia** [comment]
I agree the the ultimate power is to fork -- this code, and open source licenses, already give and preserve that power/freedom. There's no legitimate need to yield the power of maintenance/ownership of any particular, pre-existing fork (including the 'main' or 'parent' project) to others due to any real or perceived barrier to contribution to that fork.
